app/models/agents/commander_agent.rb

```diff
@@ -17,7 +17,7 @@ class CommanderAgent < Agent
 
       * `enable`: Target Agents are enabled (if not) when this agent is triggered.
 
-      * `configure`: Target Agents have their options updated with the contents of `configure_options`.
+      * `configure`: Target Agents have their options updated with the contents of `configure_options`. Set `configure_options_nested_array_update_mode` to `merge`(default) or `replace` when configuring arrays that are nested within the `configure_options` hash.
```

Comment:
When this is combined with my other PR(https://github.com/cantino/huginn/pull/1147), those depending on the deprecated(array) trigger commander agent options [and thus that behavior] will still encounter the same "replace" logic they are used to.

In this case, we are also protecting newcomers, like myself, from accidental data loss while they are still learning the intricacies of this rather complex software.